pkg/utils/mathutil/mathutil.go

```diff
@@ -49,3 +50,17 @@ func Avg[V constraints.Integer](arr ...V) (V, error) {
 
 	return total / V(len(arr)), nil
 }
+
+// Med mirrors implementation with generics: https://github.com/montanaflynn/stats/blob/249b5aaa10484bb7e8f3b866b0925aaebdac8170/median.go#L6
+func Med[V constraints.Integer](arr ...V) (V, error) {
```

Comment:
Is `Med` a typical shorthand for `Median`? The clarity seems worth a few more characters if this isn't already a well known name:
```suggestion
// Median mirrors implementation with generics: https://github.com/montanaflynn/stats/blob/249b5aaa10484bb7e8f3b866b0925aaebdac8170/median.go#L6
func Median[V constraints.Integer](arr ...V) (V, error) {
```

Reply:
fair point - i was trying to be too clever with 3 char names 🤦🏼